source/jormungandr/jormungandr/street_network/kraken.py

```diff
@@ -139,6 +141,11 @@ def _direct_path(
 
         return response
 
+    def _hanlde_car_no_park_modes(self, mode):
```

Comment:
I admit that I don't know why this rule.
We handle all car modes as a **car_no_park_mode** for _kraken_

Reply:
that's the evil part of this PR...

For kraken, when the mode is one of car/taxi/ridesharing, **car_no_park** should be used

For asgard, when the mode is one of car/taxi/ridesharing/car_no_park, **car** should be used, because asgard don't know how to park the car...